Player/player.tscn

```diff
@@ -15,5 +24,26 @@ scale = Vector2(0.96875, 0.992188)
 texture = ExtResource("2_uka3s")
 
 [node name="CollisionShape2D" type="CollisionShape2D" parent="."]
-position = Vector2(60, 78)
+position = Vector2(60, 88.4375)
 shape = SubResource("RectangleShape2D_milye")
+
+[node name="Hurtbox" type="Area2D" parent="."]
+unique_name_in_owner = true
+collision_layer = 0
+collision_mask = 2
```

Comment:
If we're going to use collision layers then they'll need to be named or it will be difficult to keep track of them.